docs/excel/excel-add-ins-pivottables.md

```diff
@@ -175,6 +176,67 @@ The following code samples changes the aggregation to be averages of the data.
     });
 ```
 
+## Change calculations with a ShowAsRule
+
+PivotTables, by default, aggregate the data of their row and column hierarchies independently. A `ShowAsRule` changes the data hierarchy to output values based on other items in the PivotTable.
+
+The `ShowAsRule` object has three properties:
+-	`calculation`: The type of relative calculation to apply to the data hierarchy (the default is `none`).
+-	`baseField`: The field within the hierarchy containing the base data before the calculation is applied. The `PivotField` usually has the same name as its parent hierarchy.
+-	`baseItem`: The individual item compared against the values of the base fields based on the calculation type. Not all calculations require this field.
+
+The following example sets the calculation on the **Sum of Crates Sold at Farm** data hierarchy to be a percentage of the column total. 
+We still want the granularity to extend to the fruit type level, so we’ll use the **Type** row hierarchy and its underlying field. 
+The example also has **Farm** as the first row hierarchy, so the farm total entries display the percentage each farm is responsible for producing as well.
+
+![A PivotTable showing the percentages of fruit sales relative to the grand total for both individual farms and individual fruit types within each farm.](../images/excel-pivots-showas-percentage.png)
+
+``` TypeScript
+    await Excel.run(async (context) => {
```

Comment:
Code block should be left-aligned within the `TypeScript` box -- i.e., move this entire code block to the left by 4 spaces so that it's left-aligned.

Suggest that you verify correct alignment for all code samples in this article.  For example:
- The code samples in this section is aligned properly: [Create a PivotTable](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-pivottables?branch=AlexJ-PivotShowAs#create-a-pivottable).
- The code samples in this section have 4-too-many leading spaces: [Change aggregation function](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-pivottables?branch=AlexJ-PivotShowAs#change-aggregation-function).

(These are just a couple of examples -- suggest that you verify spacing for all code samples in this article and make any necessary adjustments.)